### PR TITLE
[job feed page frontend] - added sort job posts by date

### PR DESCRIPTION
**Pull request by jakefarrell0**

### CODE REVIEW

**Review by rdonati — APPROVED** [review]
Does what it needs to, and the code is well written 👍

File: frontend/src/pages/JobFeed/JobFeed.tsx [hunk 1]
```diff
@@ -24,12 +26,37 @@ export default function JobFeed() {
     },
   })
   const drawerWidth = 270
+
+  //sorts data ascending or descending by createdDate 
+  function handleChange(){
+    if (selectedSort === "asc"){
+      setselectedSort("des")
```
Comment (CosineP) [inline frontend/src/pages/JobFeed/JobFeed.tsx:33]: If you make this an enum, not a string, TypeScript can help you make sure you use it right. You can even ensure you've covered every case.

Comment (CosineP) [reply frontend/src/pages/JobFeed/JobFeed.tsx:33]: Also we should think about when we have multiple sorts how we want to represent that. Presumably enum for which sort and an enum for asc dec?

File: frontend/src/pages/JobFeed/JobFeed.tsx [hunk 2]
```diff
@@ -24,12 +26,37 @@ export default function JobFeed() {
...
     // 64px offset is to account for the navbar
     <Box height={'calc(100vh - 64px)'} sx={{ ml: `${drawerWidth}px` }}>
       <FilterDrawer width={drawerWidth} />
       <Box p={3} display='flex' flexDirection='column' height='100%'>
+        <Box>
+            <Typography variant='button'>
+                {"Sort by Date: "}
+            </Typography>
+            <IconButton onClick={() => handleChange()}>
+                {getIcon(selectedSort)}
```
Comment (CosineP) [inline frontend/src/pages/JobFeed/JobFeed.tsx:57]: Neat!

File: frontend/src/pages/JobFeed/JobFeed.tsx [hunk 3]
```diff
@@ -24,12 +26,37 @@ export default function JobFeed() {
     },
   })
   const drawerWidth = 270
+
+  //sorts data ascending or descending by createdDate 
+  function handleChange(){
+    if (selectedSort === "asc"){
+      setselectedSort("des")
+      data.sort((a,b) => a.createdDate.localeCompare(b.createdDate))
```
Comment (CosineP) [inline frontend/src/pages/JobFeed/JobFeed.tsx:34]: Oh nice locale appropriate and everything.

**Review by CosineP — APPROVED** [review]

**Review by swjeong0825 — APPROVED** [review]
Great job! I like this change.


One small feedback is that maybe we can later change the toggle button from the arrow to the actual word, but I think we can deal with this later in different PR if you don't want to include that in this PR.

**Pull request merged**